_data/domains.yml

```diff
@@ -201,6 +201,12 @@ websites:
       tfa: No
       twitter: HeartInternet
 
+    - name: Hexonet
+      url: http://www.hexonet.net
```

Comment:
This is the only documentation link about it. The link itself is located under the authentication form.
![2016-10-17_12-17-48](https://cloud.githubusercontent.com/assets/1226001/19445788/c71355d6-9463-11e6-8341-c4c65e3c93b2.png)